serving/src/main/java/feast/serving/controller/ServingServiceGRpcController.java

```diff
@@ -92,6 +106,10 @@ public void getBatchFeatures(
       GetBatchFeaturesRequest request, StreamObserver<GetBatchFeaturesResponse> responseObserver) {
     try {
       RequestHelper.validateBatchRequest(request);
+      // Authorize only valid request, Get the project from 1st feature.
+      // "Project" remains same in the request for all features.
```

Comment:
Does the comment still reflect the code? It seems to check all the `projects` in the request now.

Reply:
Has been removed.

Reply:
has been removed.